### DISCUSSION TITLE
Split with Expense and Transfer

### DISCUSSION THREAD

**Discussion by floli** [firefly-iii/firefly-iii#8862] [General]
Hello,

here, supermarkets allow to withdraw money (e.g. 100 €), additionally to your purchase (e.g. 20 €). That produces one bank transfer of an expense of 120 €.

What is the correct way to model that in Firefly III? I tried to split that transaction into one part going to the Supermarket, one part going to my wallet. But Firefly does not allow that (message is that no valid target account can be found for the latter split). I suspect the problem is that this would be transfer from my account to my wallet, and not an expense.

How to tackle that situation?

Thanks!

**Comment by JC5** [comment]
This is not possible I'm afraid, you'll have to make two transactions.

**Reply by floli** [reply]
Thanks for your reply! Are there any plans to lift that restriction? It would be nice both from a UX perspective (changing of amount, adding a second transfer) and from a correctness perspective (on the list of statements it is shown as one transaction). 
Best Thanks!

**Comment by Ezwen** [comment]
Just in case, I would also be interested to do exactly that!
